### PR TITLE
Add widget specific source icons

### PR DESCRIPTION
**Pull request by gettinToasty**
widget_icons

### CODE REVIEW

File: app/components/SourceSelector.vue.ts [hunk 1]
```diff
@@ -12,6 +12,42 @@ import SlVueTree, {
...
 
+const widgetIconMap = {
+  'Alert Box': 'fas fa-bell',
+  'Stream Boss': 'fas fa-gavel',
+  'Event List': 'fas fa-th-list',
+  'The Jar': 'fas fa-beer',
+  'Stream Labels': 'fas fa-file-alt',
+  'Donation Ticker': 'fas fa-ellipsis',
+  'Chat Box': 'fas-fa-comments',
+  'Viewer Count': 'fas fa-eye',
```
Comment (holiber) [inline app/components/SourceSelector.vue.ts]: we use the same icon for the visible/invisible scene items, how it's looking now?

File: app/components/SourceSelector.vue.ts [hunk 2]
```diff
@@ -126,6 +168,7 @@ export default class SourceSelector extends Vue {
   }
 
   makeActive(treeNodes: ISlTreeNode<ISceneItemNode>[], ev: MouseEvent) {
+    console.log(this.nodes);
```
Comment (holiber) [inline app/components/SourceSelector.vue.ts]: console.log

File: app/components/SourceSelector.vue.ts (hunk 1 above)
Comment (gettinToasty) [reply app/components/SourceSelector.vue.ts]: it's actually a slightly different icon, let me post a screenshot

Comment (holiber) [reply app/components/SourceSelector.vue.ts]: Oh, I see. We used the same icon before redesign but now it's different

**Pull request merged**